### PR TITLE
#224: add recent performance to profile page

### PR DESCRIPTION
**Pull request by SergeyKhval**

### CODE REVIEW

**Comment by abeninski** [comment]
Can you put a screenshot in the PR pls

File: src/components/player/PlayerLeague.vue [hunk 1]
```diff
@@ -133,6 +165,19 @@ export default class PlayerLeague extends Vue {
   get isRanked() {
     return this.modeStat.rank > 0;
   }
+
+  get lastTenMatchesPerformance(): string {
```
Comment (Cepheid-UK) [inline src/components/player/PlayerLeague.vue:169]: Type needs to be `string[]`, otherwise it gives a type return error.

**Comment by Cepheid-UK** [comment]
Looks great, can you make it support 2v2 AT as well please if possible?

2v2ATsupport

File: src/components/player/PlayerLeague.vue (hunk 1 above)
Comment (SergeyKhval) [reply src/components/player/PlayerLeague.vue:169]: Sure, will fix

**Comment by SergeyKhval** [comment]
@Cepheid-UK API does not return data for 2vs2 AT. I am not quiet familiar with c#, so I wont touch stats service. If it's fixed, I guess recent performance will just work then

**Comment by Cepheid-UK** [comment]
I realised after I posted that comment that the 2v2 "Recent Performance" includes AT games I believe?

I'm happy with how it is anyway. Nice work, I'll merge.

**Pull request merged**